### PR TITLE
📘 Blog/CanCanCan Authorizations

### PR DESCRIPTION
**Pull request by mahabubx7**
### 📘 Blog/CanCanCan Authorizations

This project is associated with me (@mahabubx7) and my coding partner @prius2055 
- [x] No linters error
- [x] Maintained git, github workflows
- [x] Installed `CanCanCan` gem
- [x] Implemented required tasks with authorizations

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -33,6 +33,18 @@ def create
     end
   end
```
Comment (Ismailco) [inline app/controllers/posts_controller.rb:31]: - [x] Kindly make sure to use Post authorization in the Post Controller, e.g 
```ruby
 class PostsController < ApplicationController
    load_and_authorize_resource

    ...
  end
```

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @mahabubx7, @prius2055, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (mahabubx7) [reply app/controllers/posts_controller.rb:35]: I'm sorry. Can you please specify that in a little more details

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @mahabubx7, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**